### DISCUSSION TITLE
The second exercise for Day 2 morning (Health Statistics) doesn't have a solution

### DISCUSSION THREAD

**Discussion by njr0** [google/comprehensive-rust#1239] [General]
As far as I can see, the course doesn't include a solution for the day 2 morning exercise.

Or at least, if it does, it isn't linked from the right-arrow on the Library exercise.

(The link from http://localhost:3000/exercises/day-2/solutions-morning.html points to http://localhost:3000/exercises/day-2/solutions-afternoon.html.

(FWIW, I made the tests pass without too many problems. But I like to look at the model solutions to see if there was a better way; or if the tests didn't capture everything!)

**Comment by mgeisler** [comment]
Hey @njr0, this is actually being written by @fw-immunant in #1172.

**Reply by njr0** [reply]
Ah, OK. I have a working solution, but it might not be a model solution! I'll look at 1172. Should I close this? (Or do you want to?)

**Reply by mgeisler** [reply]
> Should I close this? (Or do you want to?)

I don't think discussions can be closed... I guess they just stick around :slightly_smiling_face: I could apparently have converted it to an issue, but I didn't think of it — I'm grateful for all feedback, regardless of the form! :smile:

**Reply by njr0** [reply]
I guess it's a form of immortality...albeit, a lesser form than the "not dying" kind 😄.

**Reply by mgeisler** [reply]
Yeah, I think we'll have to take what we get here :smile: The solution is now merged, thanks for reminding me and @fw-immunant about it.
